tests/core/performance/test_perf_tests.py

```diff
@@ -0,0 +1,28 @@
+"""
```

Comment:
Not sure about putting this file here.

It's not exactly a test of the same nature that all the unit tests that live around here, and we definitely don't want to run this in the CI.

Reply:
Seems like we are more leaning towards moving everything to France anyway.

But to answer your question, I'm not sure. I like the idea of `tests` behind executable tests that run in the CI, and crash if there is an issue. Which the performance test is not, at least in its current state. @MattiSG, any insight on good practice about where to put a performance measurement script?

Reply:
This is called _continuous inspection_ and is usually driven by metadata derived from test runs. Other metadata that is usually extracted are coverage and churn.